diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index 869a6be..64b966f 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -1,7 +1,9 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.EXPANSION_MAX_LENGTH = exports.EXPANSION_MAX = void 0;
-exports.expand = expand;
+module.exports = expand;
+module.exports.expand = expand;
+module.exports.default = expand;
 const balanced_match_1 = require("balanced-match");
 const escSlash = '\0SLASH' + Math.random() + '\0';
 const escOpen = '\0OPEN' + Math.random() + '\0';
